.github/workflows/noetic_gencpp.yml

```diff
@@ -0,0 +1,32 @@
+name: Noetic gencpp
```

Comment:
Probably a TODO for later, but after getting more experience with github ci, seems like we should have just one workflow with each of these as seperate jobs in one file?

Reply:
I'm not sure about best practices here either. Will look into after I get features merged.